tests/checkers/unittest_deprecated.py

```diff
@@ -91,6 +91,31 @@ def deprecated_method():
         ):
             self.checker.visit_call(node)
 
+    def test_deprecated_method_alias(self):
+        # Tests detecting deprecated method defined as alias
+        # to existing method
+        node = astroid.extract_node(
+            """
+        class Deprecated:
+            def _deprecated_method():
+                pass
+
+            deprecated_method = _deprecated_method
```

Comment:
Could you also add a functional test so it's possible to check that the warning is on the proper line ? Right now we're checking that something is raised for the whole node without checking the line and column.

Reply:
I've done it because I want your feature to be in 2.9.0 :)